### PR TITLE
adds button component and card details component, combines mobile and…

### PR DESCRIPTION
**Pull request by gibstock**
… desktop layout for featured destinations component

### CODE REVIEW

File: components/home/exploreTheWorld/cardDetail.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const DetailedCard = ({price, location, rooms, reviews, reviewAmount, imgPath, hotelName, distanceToTownCenter}: AppProps) => {
+  return (
+    <>
+      <div className="card-wrapper px-4 pt-4 pb-7 bg-white rounded-2xl">
+        <div className="content-wrapper flex flex-col p-0">
+          <div className="top-half flex flex-col justify-center items-start gap-5">
+            <div className="image w-60 h-40 rounded-2xl bg-cover" style={{backgroundImage: `url(${imgPath})`}}></div>
+            <div className="flex flex-row items-center gap-2">
+              <div className="star-icon">
+                <img src="/star.png" alt="star for rating" />
```
Comment (PeterSYoo) [inline components/home/exploreTheWorld/cardDetail.tsx:23]: It seems the problem stems from the img tag, I believe Next requires us to use its own built in Image api.

File: components/home/exploreTheWorld/cardDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+              <div className="flex flex-col justify-start items-start gap-1">
+                <span className="font-medium text-xl text-[#3b3e44]">{hotelName}</span>
+                <span className="font-medium text-sm text-[#84878b]">{distanceToTownCenter} km to Town Center</span>
+              </div>
+              <Button>${price}</Button>
+            </div>
+            <div className="row2 flex flex-col justify-center items-start gap-2">
+              <div className="flex flex-row justify-space-between items-center gap-2">
+                <div className="icon">
+                  <img src="/gps-marker.png" alt="gps marker" />
```
Comment (PeterSYoo) [inline components/home/exploreTheWorld/cardDetail.tsx:39]: Similar issue here.

File: components/home/exploreTheWorld/cardDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                </div>
+                <span className='font-normal text-sm text-[#84878B]'>{location}</span>
+              </div>
+              <div className="flex flex-row justify-space-between items-end gap-2">
+                <div className="icon">
+                  <img src="/building-icon.png" alt="building icon" />
```
Comment (PeterSYoo) [inline components/home/exploreTheWorld/cardDetail.tsx:45]: Similar issue here.

Comment (gibstock) [reply components/home/exploreTheWorld/cardDetail.tsx:45]: That's a strange requirement. However an easy fix.

**Pull request closed without merging**